forest_benchmarking/qubit_spectroscopy.py

```diff
@@ -50,58 +64,144 @@ def generate_t1_experiments(qubits: Union[int, List[int]],
     :param stop_time: The maximum decay time to measure at.
     :param n_shots: The number of shots to average over for each data point.
     :param num_points: The number of points for each t1 curve.
-    :return: list of tuples in the form: (time, t1 program with decay of that time)
+    :return: A dataframe with columns: time, t1 program
     """
     start_time = 0
     time_and_programs = []
-    for t in np.linspace(start_time, stop_time, num_points):
+
+    for t in np.linspace(start_time, stop_time, n_points):
         t = round(t, 7)  # try to keep time on 100ns boundaries
-        time_and_programs.append((t, generate_single_t1_experiment(qubits, t, n_shots)))
-    return time_and_programs
+        time_and_programs.append({
+            'times': t,
```

Comment:
RB convention would be "Time" and "Program", i.e. singular and capitalized.

Reply:
Good point. I did this and made all the other fields in the DataFrame capitalized and singular.